deploy-board/deploy_board/static/js/components/clusterconfigcomponents.js

```diff
@@ -595,8 +595,9 @@ Vue.component('backup-hosttypes', {
         },
         backupTypesWarning: function () {
             const part1 = "You should only enable this feature with capacity-eng approval. ";
-            const part2 = "Your enable/disable operation on this feature triggers a notification to capacity-eng.";
-            const text = part1 + part2;
+            const part2 = "Your enable/disable operation on this feature triggers a notification to capacity-eng. ";
+            const part3 = "For all the instance type mapping information, please go to https://deploy.pinadmin.com/clouds/hosttypesmapping";
```

Comment:
The url take a little while to load by the way

Reply:
Also, should this be a link (e.g. `<a>` tag)?

Reply:
in the warning, we could not add <a> in it